dhall/src/Dhall/Main.hs

```diff
@@ -210,6 +213,14 @@ parseOptions =
             <>  Options.Applicative.help description
             )
 
+    flag name description =
+        Options.Applicative.flag
+            (Just True)
```

Comment:
@Gabriel439 I'm pretty sure we want this one to default to `Just False`, since the defaults for both `--ascii` and `--explain` are `False`